service/controller/v21/version_bundle.go

```diff
@@ -7,25 +7,35 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "calico",
+				Description: "Update calico to 3.6.1.",
+				Kind:        versionbundle.KindChanged,
+			},
 			{
 				Component:   "cloudconfig",
 				Description: "Pin calico-kube-controllers to master.",
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "containerlinux",
+				Description: "Update to 2023.5.0.",
+				Kind:        versionbundle.KindChanged,
+			},
 			{
 				Component:   "kubernetes",
 				Description: "Update kubernetes to 1.14.1. More info here: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md",
 				Kind:        versionbundle.KindChanged,
 			},
 			{
-				Component:   "calico",
-				Description: "Update calico to 3.6.1",
-				Kind:        versionbundle.KindChanged,
+				Component:   "kubernetes",
+				Description: "Tolerate all taints for calico and kube-proxy daemon sets.",
+				Kind:        versionbundle.KindFixed,
 			},
 			{
-				Component:   "containerlinux",
-				Description: "Update to 2023.5.0.",
-				Kind:        versionbundle.KindChanged,
+				Component:   "kvm-operator",
+				Description: "Go back to XFS for docker and kubelet volumes.",
```

Comment:
```suggestion
				Description: "Use XFS for docker and kubelet volumes.",
```
Makes it sound less flippy-floppy :D

Reply:
Because it is :)